application-admintools-default/src/main/java/com/xwiki/admintools/internal/data/identifiers/TomcatIdentifier.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.admintools.internal.data.identifiers;
+
+import java.io.File;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+
+/**
+ * Encapsulates functions used for identifying a Tomcat server and retrieving it's info.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Component
+@Named(TomcatIdentifier.HINT)
+@Singleton
+public class TomcatIdentifier extends AbstractServerIdentifier
+{
+    /**
+     * Component identifier.
+     */
+    public static final String HINT = "tomcat";
+
+    /**
+     * Verify if this server is the one that stores the XWiki instance.
+     *
+     * @return true if this server is used, false otherwise
+     */
+    @Override
+    public boolean isUsed(String providedConfigServerPath)
+    {
+        if (tomcatIsUsed(providedConfigServerPath)) {
+            updatePaths(providedConfigServerPath);
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    @Override
+    public String getIdentifier()
+    {
+        return HINT;
+    }
+
+    @Override
+    public void updatePaths(String providedConfigServerPath)
```

Comment:
In case it is not automatically found, it should be checked again if the user changed the configuration. There were some useless calls tho, and I did refactor these functions.

Reply:
Right, it cannot return `usedServer != null` with the wrong server path. If the user inserts a wrong configuration, nothing will happen until a server restart. I could return a boolean to the UI if the configuration path is is introduced correct or not and throw a warning or error message  in case it's not?